src/index.js

```diff
@@ -0,0 +1,89 @@
+import './style.css';
+import { addLikeEvent, getAllLikes } from './modules/likesCounter.js';
+import { createApp } from './modules/movies.js';
+import { postComment, getComments } from './modules/comments.js';
+import { fetchMovies } from './modules/moviesCounter.js';
+
+createApp();
+
+const container = document.querySelector('.movies_container');
+
+const populateUI = (data, likes) => {
+  let like = likes.filter((like) => like.item_id === String(data.id));
+  like = like.length ? like[0].likes : 0;
+
+  const mainDiv = document.createElement('div');
+  mainDiv.classList.add('main_card');
+  const div = document.createElement('div');
+  div.classList.add('movie_wrapper');
+  div.innerHTML = `
+    <div class="poster_wrapper">
+        <img src=${data.image.medium} alt="">
+        <div class="details_wrapper">
+          <div class="paragraph_container">
+            <p>${data.name}</p>
+            <span>HD</span>
+          </div>
+          <div class="likes">
+
+            <div id=${data.id} class="likes_wrap">
+                <span class="counter">${like}</span>
+                <img class="heart" src="https://img.icons8.com/color/48/ffffff/hearts.png"/>
+            </div>
+          </div>
+        </div>
+    </div>
+    <div class="buttons_wrapper" id=${data.id}>
+        <button id=${data.id} class="comment">Comments</button>
+    </div>
+    `;
+  mainDiv.appendChild(div);
+  return mainDiv;
+};
+
+const popup = async (id, likes) => {
+  let results = await fetch(`https://api.tvmaze.com/shows/${id}`);
+  localStorage.setItem('id', id);
+  let like = likes.filter((like) => like.item_id === String(id));
+  like = like.length ? like[0].likes : 0;
+  results = await results.json();
+  document.querySelector('.popup').style.display = 'flex';
+  document.querySelector('.displaying').setAttribute('src', results.image.medium);
+  document.querySelector('.like_counter').innerText = `${like}`;
+  document.querySelector('.image-arrow').addEventListener('click', () => {
+    document.querySelector('.popup').style.display = 'none';
+  });
+  getComments();
+  document.querySelector('.movie_name').innerText = results.name;
+};
+
+const comment = async () => {
+  const comments = document.querySelectorAll('.buttons_wrapper');
+  const likes = await getAllLikes();
+  comments.forEach((element) => {
+    element.addEventListener('click', () => {
+      const id = element.getAttribute('id');
+      popup(id, likes);
+    });
+  });
+};
+
```

Comment:
- [ ]  You project should contain comments that matches their relative item.  When I try to comment on a specific item, and click to the other item's comment section, I am getting previous comment to the new item as well. Try to fix it , so that every time a user try to comment, on a specific movies, it will not appear to the other movies as well. 
Added comments to this movies 
![image](https://user-images.githubusercontent.com/43877689/218257105-0916b435-8f12-4dfd-b0f2-82847afea9fb.png)
The other movies have the same comments 
![image](https://user-images.githubusercontent.com/43877689/218257134-11a5b232-d7bd-4174-af9f-2a7d2fc0a4fa.png)

Reply:
@ishimwezachee Hi, the comments take time to load that's why at first they show up but the page gets to load the right comments after loading properly. if there’s anything that you'd suggest to make the page load faster please let us know.

Reply:
Hi @ishimwezachee we are solving it thank you.